stellarphot/core.py

```diff
@@ -201,6 +202,19 @@ def __repr__(self):
         )
 
 
+# Add YAML round-tripping for Camera
```

Comment:
So it looks like AstropyDumper handles conversion of objects to their YAML equivalents, interesting.

Reply:
However, I am just trying to understand the logic here (and inline documentation on this might help us recall this in the future if we choose to add more to these custom AstroPy tables). 

`camera_representer` and `camera_constructor` are currently implemented as functions created when we import core.  From the astropy documentation it seems "[AstropyDumper] is not directly instantiated by user code, but instead is used to maintain the available [representer functions that are called when generating a YAML stream from an object." and AstropyLoader is similar from the constructor end.  It looks like they derived from PyYAML.  Can you clarify why "!Camera" is used in `camera_representer` and AstropyLoader?

I don't see a place where these are actually *implemented*` in `PhotometryData` so I am guessing on any astropy table (or in our case, the subclasses of them) we import/export to ECSV will use AstropyLoader and AstropyDumper to handle the attributes of the table... in which case, do you need similar representers/constructors for `observatory`?